PyInstaller/utils/hooks/__init__.py

```diff
@@ -16,7 +16,7 @@
 import re
 
 from ...compat import base_prefix, exec_command_stdout, exec_python, \
-    is_darwin, is_py2, is_py3, is_venv, string_types, open_file, \
+    is_darwin, is_py2, is_py3, is_venv, is_conda, string_types, open_file, \
```

Comment:
Not used here? Please undo.

Reply:
Ah yes, this was when I assumed this module was imported automatically for all hooks. Forgot to remove it.